api/src/test/java/uk/gov/dhsc/htbhf/claimant/ClaimantServiceAssertionUtils.java

```diff
@@ -36,10 +39,10 @@ public static void assertClaimantMatchesClaimantDTO(ClaimantDTO claimant, Claima
     }
 
     // TODO HTBHF-2705 Change this to hardcoded v3 claim endpoint instead of a URI parameter.
```

Comment:
probably want to remove this comment too

Reply:
Removed.